R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -10,6 +10,23 @@ vignette: >
   \usepackage[utf8]{inputenc}
 ---
 
+<!--
```

Comment:
right, this file actually contains source code that gets run to generate output that gets included (as a html file) in the Apache source release.